aamg/generator.py

```diff
@@ -0,0 +1,139 @@
+import argparse
+import random
+import typing
+
+import log
+
+
+class ModelGenerator:
+    '''This is the main handle for our generator'''
+
+    def __init__(self, args: argparse.Namespace) -> None:
+        '''Initialize our handle'''
+        self.args = args
+        self.parse_grammar()
+        self.load_assets()
+
+    def parse_grammar(self) -> None:
+        '''Parse grammar file'''
+        if self.args.verbose:
+            log.info(f'loading grammar file: {self.args.grammar}')
+        try:
+            with open(self.args.grammar, 'r') as f:
+                lines = (line.strip().split('=') for line in f.readlines())
+        except OSError as e:
+            log.die(f'{self.args.grammar}: {e.strerror}')
+        self.grammar: typing.Dict[str, str] = dict()
```

Comment:
Should this be typing.Dict[str, typing.List[str]] ?

Reply:
yup ! gonna fix this.